nvidia/cheminformatics/data/helper/chembldata.py

```diff
@@ -12,17 +11,17 @@
 from contextlib import closing
 from nvidia.cheminformatics.utils.singleton import Singleton
 from nvidia.cheminformatics.smiles import RemoveSalt, PreprocessSmiles
-from nvidia.cheminformatics.fingerprint import MorganFingerprint, Embeddings
+from nvidia.cheminformatics.fingerprint import Embeddings
 
 SMILES_TRANSFORMS = [RemoveSalt(), PreprocessSmiles()]
 FINGERPRINT_SELECTION = Embeddings
```

Comment:
Fingerprint selection should be changed to MorganFingerprint to use that by default

Reply:
This changes my previous comment

Reply:
@rilango My understanding is then that Embeddings were the default? Regardless, I trust you to fix the issue if it does exist. You can make any needed changes (or not) and merge the code when you're ready. You should have merge rights.

Reply:
@mlgill Please check this new commit to cleanup this part of code further https://github.com/NVIDIA/cheminformatics/pull/17/commits/dd6e710b82b03af3babe52bb89a67499d4763d64

This commit also had changes to use properties file to change configuration.